src/rx_signal_processing.py

```diff
@@ -220,6 +220,15 @@ def sequence_worker(options, ringbuffer):
             )
             cfs_processor.apply_filters(sequence_samples)
             cfs_processor.move_filter_results()
+            total_phases = np.empty(
+                (rx_params.main_beam_angles.shape[0], 1, 20), dtype=np.complex64
```

Comment:
I would recommend against magic numbers, see where you can get the total number of antennas from